test/test_misc.py

```diff
@@ -1,8 +1,12 @@
+import re
+import string
 import sys
+import textwrap
 
 import pytest
+from IPython.lib.pretty import pretty
 
-from galgebra.ga import OrderedBiMap
+from galgebra.ga import lazy_dict, OrderedBiMap, GradedTuple
```

Comment:
Old question, can they be somewhere other then in `ga.py`?

Reply:
Definitely - can you suggest a name?

Reply:
`data` is not too bad.